srcSpecs/demo-spec.js

```diff
@@ -0,0 +1,17 @@
+var test = require('tape');var file = require('../src/demo.js');test('sum function', function(t) { t.plan(2); 
```

Comment:
I'm assuming we have this file in the PR so that we can work out the formatting issues. Is this correct?